kernel/build-kernel.sh

```diff
@@ -136,19 +138,93 @@ get_major_kernel_version() {
 	echo "${major_version}.${minor_version}"
 }
 
+# Make a kernel config file from generic and arch specific
+# fragments
+# - arg1 - path to arch specific fragments
+# - arg2 - path to kernel sources
+#
+get_kernel_frag_path() {
+	local arch_path="$1"
+	local common_path="${arch_path}/../common"
+	local kernel_path="$2"
+	local cmdpath="${kernel_path}/scripts/kconfig/merge_config.sh"
+	local config_path="${arch_path}/.config"
+
+	local arch_configs="$(ls ${arch_path}/*.conf)"
```

Comment:
Do we need arch-specific kernel fragment config files for all the other (non-`x86_64`) architectures to ensure those kernel continue to build as they do today?

Reply:
The code is written so if it does not find arch specific fragments it reverts to using the non-fragment configs. Thus, we can add arches as and when they have their fragments ready, and not delay the merge of the framework and x86 fragments :-)